R/api.R

```diff
@@ -1,4 +1,4 @@
-build_api <- function(runner, path, rate_limit = 2 * 60) {
+build_api <- function(runner, path, backup, rate_limit = 2 * 60) {
```

Comment:
What is your thinking here for passing in a backup object rather than the period alone? We construct the api object so it feels like it'd be more symmetric to accept the backup args here?

Reply:
Thanks yeah, was basing this on runner which we construct before passing it to `build_api`. I switched this out though to pass `backup_period` and it leads to way less churn